### ISSUE TITLE
Linux Support

### ISSUE THREAD

**Issue by ppk-gh** [BadDogSkyrim/PyNifly#75]
While blender runs natively, this plugin seems to only support Windows. I just wanted to know whether you plan to support other platforms in the future. nif -> OS -> obj -> Blender and the reverse work but there are a few inconveniences.

**Comment by ppk-gh** [comment]
From what I can tell the python part works but the .dll bit is windows specific code. The NifTools blender plugin works though it has its own issues.

**Comment by BadDogSkyrim** [comment]
Of course, duh. Okay, not something I can get into, at least not in the near term.
